appserver/connectors/admin/src/main/java/org/glassfish/connectors/admin/cli/FlushConnectionPoolLocal.java

```diff
@@ -67,23 +68,28 @@
 import org.glassfish.config.support.CommandTarget;
 import org.glassfish.config.support.TargetType;
 
-@Service(name = "flush-connection-pool")
+/**
+ * The actual command to flush a connection pool.
+ * This will flush a specified connection pool on
+ * this instance.
+ * @see FlushInstancesConnectionPool
+ */
+@Service(name = "_flush-connection-pool")
 @PerLookup
-@I18n("flush.connection.pool")
 @TargetType(value = {CommandTarget.DOMAIN, CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.DEPLOYMENT_GROUP,
     CommandTarget.CLUSTERED_INSTANCE})
 @ExecuteOn(value = {RuntimeType.ALL})
 @RestEndpoints({
     @RestEndpoint(configBean=Resources.class,
         opType=RestEndpoint.OpType.POST, 
-        path="flush-connection-pool", 
+        path="_flush-connection-pool", 
         description="flush-connection-pool")
 })
-public class FlushConnectionPool implements AdminCommand {
-    final private static LocalStringManagerImpl localStrings =
-            new LocalStringManagerImpl(FlushConnectionPool.class);
+public class FlushConnectionPoolLocal implements AdminCommand {
+    
+    final private static LocalStringManagerImpl localStrings = new LocalStringManagerImpl(FlushConnectionPoolLocal.class);
 
-    @Param(name = "pool_name", primary = true)
+    @Param(name = "poolName")
```

Comment:
What's the reasoning behind not making this the primary param?

Reply:
Because then it doesn't work when RemoteCLICommand tries to call it as it sends a ParameterMap of key-value pairs, a lone value is not accepted.